src/lib/hash/keccak/info.txt

```diff
@@ -4,6 +4,7 @@ KECCAK -> 20131128
 
 <module_info>
 name -> "Keccak"
+lifecycle -> "Deprecated"
```

Comment:
Perhaps retroactively add a 'brief' explaining that this is _not_ SHA-3. It did cause plenty confusion in the past, so repeating this a few times surely won't hurt... 🙂